README.md

```diff
@@ -94,6 +94,10 @@ JDK17 is required to build. Ensure that you have JAVA_HOME set properly.
 * `./gradlew test`: Run the tests
 * `./gradlew mavenZip`: Create a zip file that can be uploaded to Maven Central. VERSION, SIGNING_KEY, and SIGNING_PASSWORD envinroment variables must be provided.
 
+### Releasing
+There is a GitHub Action that runs the mavenZip target and attaches the result to any release created via the GitHub UI. 
+This ZIP file must be manually uploaded to Maven Central. 
```

Comment:
What is the github action called?

Reply:
Why does that matter?

Reply:
Because the documentation doesn't say what launches it, and I want to make sure I know how (or if) I need to initiate that process.

Reply:
Oh. I see the confusion. I'll change the wording. The github action is launched automatically when you create the release.